api/src/services/users/users.js

```diff
@@ -1,9 +1,9 @@
 import { db } from 'src/lib/db'
 
 // Only to be used on the api side
```

Comment:
Strategy for limiting functionality to the api side: just make sure there's no sdl for it